grpc/tests/python/MyGame/Sample/monster_grpc_fb.py

```diff
@@ -0,0 +1,190 @@
+        response_deserializer=Monster.FromString,
+        )
+
+
+class MonsterStorageServicer(object):
+  
+  def Store(self, request, context):
+    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+    context.set_details('Method not implemented!')
+    raise NotImplementedError('Method not implemented!')
+  
+  def Retrieve(self, request, context):
+    context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+    context.set_details('Method not implemented!')
+    raise NotImplementedError('Method not implemented!')
+
+
+def add_MonsterStorageServicer_to_server(servicer, server):
+  rpc_method_handlers = {
+      'Store': grpc.unary_unary_rpc_method_handler(
+          servicer.Store,
+          request_deserializer=Monster.FromString,
+          response_serializer=Weapon.SerializeToString,
+      ),
+      'Retrieve': grpc.unary_unary_rpc_method_handler(
+          servicer.Retrieve,
+          request_deserializer=Weapon.FromString,
+          response_serializer=Monster.SerializeToString,
+      ),
+  }
+  generic_handler = grpc.method_handlers_generic_handler(
+      'MyGame.Sample.MonsterStorage', rpc_method_handlers)
+  server.add_generic_rpc_handlers((generic_handler,))
+try:
+  # THESE ELEMENTS WILL BE DEPRECATED.
+  # Please use the generated *_pb2_grpc.py files instead.
+  import grpc
+  
+  import Monster_fb as Monster__fb
+  import Weapon_fb as Weapon__fb
+  from grpc.beta import implementations as beta_implementations
+  from grpc.beta import interfaces as beta_interfaces
+  from grpc.framework.common import cardinality
+  from grpc.framework.interfaces.face import utilities as face_utilities
+  
+  
+  class MonsterStorageStub(object):
+    
+    def __init__(self, channel):
+      """Constructor.
+      
+      Args:
+        channel: A grpc.Channel.
+      """
+      self.Store = channel.unary_unary(
+          '/MyGame.Sample.MonsterStorage/Store',
+          request_serializer=Monster.SerializeToString,
+          response_deserializer=Weapon.FromString,
+          )
+      self.Retrieve = channel.unary_unary(
+          '/MyGame.Sample.MonsterStorage/Retrieve',
+          request_serializer=Weapon.SerializeToString,
+          response_deserializer=Monster.FromString,
+          )
+  
+  
+  class MonsterStorageServicer(object):
+    
+    def Store(self, request, context):
+      context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+      context.set_details('Method not implemented!')
+      raise NotImplementedError('Method not implemented!')
+    
+    def Retrieve(self, request, context):
+      context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+      context.set_details('Method not implemented!')
+      raise NotImplementedError('Method not implemented!')
+  
+  
+  def add_MonsterStorageServicer_to_server(servicer, server):
+    rpc_method_handlers = {
+        'Store': grpc.unary_unary_rpc_method_handler(
+            servicer.Store,
+            request_deserializer=Monster.FromString,
```

Comment:
I tried removing the `import Monster_fb` and `import Weapon_fb` while testing, after that, this line gave an error saying that, `name Monster is not defined`.

Reply:
I think we need to change `get_input_type_name` and `get_output_type_name` in `idl_gen_grpc` to something more appropriate?

Reply:
why, what should they do? How would that affect C++/Go?

Reply:
I guess, I will have to write my own `SerializeToString` and `FromString` methods for python generator? Since those methods are protobuf specific. Are there any serializers or deserializers already written for python. I saw that C++ grpc has Serialize and Deserialize methods.

Reply:
well, what is the cleanest way to lift out the protobuf specificness? Should that happen here or in the grpc project?

Reply:
It should happen here.